style.css

```diff
@@ -0,0 +1,826 @@
+  height: 30vh;
+  background-size: cover;
+  background-repeat: no-repeat;
+  align-items: flex-end;
+  padding: 0.7rem;
+  margin-bottom: 2px;
+}
+
+.variety-news-label {
+  margin-bottom: 0.4rem;
+}
+
+.variety-news-poster .article-title {
+  margin-bottom: 0.5rem;
+}
+
+.variety-news-img-container {
+  background-image: url(https://img-cdn.tnwcdn.com/image?fit=1280%2C720&url=https%3A%2F%2Fcdn0.tnwcdn.com%2Fwp-content%2Fblogs.dir%2F1%2Ffiles%2F2019%2F08%2Fbitcoin_today-header_bitcoin_today.jpg&signature=b75eca35f42accc83310a4fa4e562088);
+  height: 96px;
+  width: 96px;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 100%;
+}
+
+.variety-news-related {
+  font-size: 14px;
+  list-style-type: none;
+  list-style-position: inside;
+}
+
+.variety-news-latest {
+  background-color: rgb(255, 247, 246);
+  color: #f42;
+}
+
+.variety-news-latest p {
+  padding: 1rem;
+  font-weight: 600;
+}
+
+.variety-news-related a::before {
+  content: "\2022";
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1rem;
+}
+
+/* latest deals */
+
+.latest-deal-header {
+  width: 100%;
+}
+
+.latest-deal-header h3 {
+  float: left;
+  margin: 0;
+  padding: 0;
+}
+
+.latest-deal-header div {
+  float: right;
+  padding: 0.5rem;
+}
+
+.latest-deals-discount {
+  font-size: 1.2rem;
+}
+
+.article-metadata span {
+  text-decoration: line-through;
+  font-size: 0.8rem;
+}
+
+.latest-deals-img-container {
+  width: 75px;
+  height: 75px;
+  background-image: url(https://cdnp2.stackassets.com/1b0bd98a9c257d0deba32330f074a986c0ac80ab/store/5244ce0f683265f98baeb686db693a0571c541750ca7a48220c02ced7df1/sale_32417_primary_image.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 50%;
+}
+
+.discount {
+  display: none;
+}
+
+/* topfooter */
+
+.top-footer {
+  background-color: #131313;
+  flex-direction: column;
+  align-items: center;
+  padding: 40px 0 30px 0;
+}
+
+.footer-brand button:hover {
+  background-color: black;
+}
+
+.footer-last-social-button {
+  margin-right: 0;
+}
+
+.footer-about-us {
+  color: #aaa;
+  font-size: 0.8rem;
+  margin-top: 32px;
+}
+
+.footer-last-info-element {
+  margin-right: 0;
+}
+
+.footer-about-us a:hover {
+  color: white;
+}
+
+/* bottomfooter */
+
+.bottom-footer {
+  flex-direction: column;
+  align-items: center;
+  background-color: black;
+  color: white;
+  font-size: 0.8rem;
+  padding: 40px 0;
+}
+
+.bottom-footer span {
+  color: #aaa;
+  margin-top: 5px;
+}
+
+/* media query */
+
+@media only screen and (min-width: 768px) {
+  /* global */
+
+  body section {
+    margin-bottom: 5rem;
+  }
+
+  /* header */
+
+  header .brand {
+    padding-top: 30px;
+  }
+
+  .navbars {
+    width: 100%;
+  }
+
+  .navbars .top-navbar {
+    padding: 0.6em 0;
+    justify-content: space-between;
+    display: flex;
+    font-size: 13px;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in the entire CSS file, because you already have in this file, line #30
- [ ] Please use it your rule
**Example:**

**Html file**

```HTML
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
The other areas where we repeated "display: flex" was done ad-hoc. It began right in the media queries for the medium screen size. So we had to use it to display and layout 3 individual elements.